knora/dsplib/schemas/data.xsd

```diff
@@ -1,43 +1,46 @@
-<?xml version="1.0"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1">
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema xmlns="https://dasch.swiss/schema"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="https://dasch.swiss/schema"
+           elementFormDefault="qualified">
```

Comment:
from the top of my head, I couldn't say if that does what it should. I assume you tested it and it worked?

Reply:
yes, I tested it